cypress/e2e/core/ui/video_test_cases

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Do we need this text file? These type of items might be better placed in Jira as tickets?